reporters/wdio/TerraWDIOTestDetailsReporter.js

```diff
@@ -309,13 +322,11 @@ class TerraWDIOTestDetailsReporter extends events.EventEmitter {
    * @return null
    */
   // eslint-disable-next-line class-methods-use-this
-  writToFile(data, filePath) {
+  writeToFile(data, filePath) {
     try {
-      fs.writeFileSync(
-        filePath,
-        `${JSON.stringify(data, null, 2)}`,
-        { flag: 'w+' },
-      );
+      fs.writeFileSync(filePath, `${JSON.stringify(data, null, 2)}`, {
```

Comment:
if we're worried there's too much noise in the parameters, we could use variables for the data and option flags.
```suggestion
      const json = `${JSON.stringify(data, null, 2)}`;
      const options = { flag: 'w+' };
      fs.writeFileSync(filePath, json, options);
```

Reply:
addressed